prow/plugins/lgtm/lgtm.go

```diff
@@ -140,17 +143,20 @@ func handleGenericComment(gc githubClient, config *plugins.Configuration, owners
 
 	// If we create an "/lgtm" comment, add lgtm if necessary.
 	// If we create a "/lgtm cancel" comment, remove lgtm if necessary.
-	wantLGTM := false
+	// If we create a "/tbr" comment, add lgtm and to-be-reviewed if necessary.
+	wantLGTM, wantTBR := false, false
 	if lgtmRe.MatchString(rc.body) {
 		wantLGTM = true
 	} else if lgtmCancelRe.MatchString(rc.body) {
 		wantLGTM = false
+	} else if tbrRE.MatchString(rc.body) {
```

Comment:
So a comment like


```
/tbr
/lgtm
```

Will only pick up need for LGTM?

Reply:
that would be mis-use of the plugin anyhow? The same is true of lgtm vs lgtm cancel currently
LGTM and TBR are mutually exclusive, if you can LGTM you can't TBR, and if you can TBR, you can't LGTM